datafusion/common/src/dfschema.rs

```diff
@@ -353,13 +353,13 @@ impl DFSchema {
 
     /// Returns an immutable reference of a specific `Field` instance selected using an
```

Comment:
Perhaps need to update some of these method docstrings since they assume returning a `Field`